### PR TITLE
Adding Getters for Source and SchemaURL

### PR DESCRIPTION
**Pull request by nachocano**
**Proposed Changes**

- Adding Source() and SchemaURL() to Event.
- Adding GetSource() and GetSchemaURL() to EventContext interface.
- Adding UTs.

### CODE REVIEW

File: pkg/cloudevents/eventcontext.go [hunk 1]
```diff
@@ -31,6 +31,12 @@ type EventContext interface {
 	// GetType returns the CloudEvents type from the context.
 	GetType() string
 
+	// GetSource returns the CloudEvents source from the context.
+	GetSource() string
+
+	// GetSchema returns the CloudEvents schema (if any) from the context.
+	GetSchema() string
```
Comment (n3wscott) [inline pkg/cloudevents/eventcontext.go]: The property is called SchemaURL

File: pkg/cloudevents/event.go [hunk 2]
```diff
@@ -31,6 +31,16 @@ func (e Event) Type() string {
 	return e.Context.GetType()
 }
 
+// Source returns Context.GetSource()
+func (e Event) Source() string {
+	return e.Context.GetSource()
+}
+
+// Schema returns Context.GetSchema()
+func (e Event) Schema() string {
```
Comment (n3wscott) [inline pkg/cloudevents/event.go]: The property is called SchemaURL

File: pkg/cloudevents/eventcontext_v02.go [hunk 3]
```diff
@@ -72,6 +72,19 @@ func (ec EventContextV02) GetType() string {
 	return ec.Type
 }
 
+// GetSource implements EventContext.GetSource
+func (ec EventContextV02) GetSource() string {
+	return ec.Source.String()
+}
+
+// GetSchema implements EventContext.GetSchema
+func (ec EventContextV02) GetSchema() string {
```
Comment (n3wscott) [inline pkg/cloudevents/eventcontext_v02.go]: The property is called SchemaURL

File: pkg/cloudevents/eventcontext.go (hunk 1 above)
Comment (nachocano) [reply pkg/cloudevents/eventcontext.go]: OK, do you prefer a GetSchemaURL or GetSchemaUrl?

What about the method in Event. Are you OK to leave it as Schema() or do you prefer a SchemaURL/SchemaUrl?

Comment (n3wscott) [reply pkg/cloudevents/eventcontext.go]: Go style is `GetSchemaURL`

Comment (n3wscott) [reply pkg/cloudevents/eventcontext.go]: because `SchemaURL` conflicts with the struct member.v

Comment (nachocano) [reply pkg/cloudevents/eventcontext.go]: Sounds good, will do, thanks!

File: pkg/cloudevents/event.go (hunk 2 above)
Comment (nachocano) [reply pkg/cloudevents/event.go]: Done

File: pkg/cloudevents/eventcontext_v02.go (hunk 3 above)
Comment (nachocano) [reply pkg/cloudevents/eventcontext_v02.go]: Done

File: pkg/cloudevents/eventcontext.go (hunk 1 above)
Comment (nachocano) [reply pkg/cloudevents/eventcontext.go]: Done

**Comment by n3wscott** [comment]
Looks good to me.

**Comment by n3wscott** [comment]
Mark to merge.

**Comment by markpeek** [comment]
LGTM! Merged.

**Pull request merged**